### ISSUE TITLE
Jupyter installation is using tornado 6 but it not work with tornado 6

### ISSUE THREAD

**Issue by timotta** [jupyter/jupyter#401]
When installing jupyter in a new environment, we cannot use the web notebooks because it cannot connect to the kernel. The following message appear on terminal, while in the notebook appears the message "Connecting to kernel" indefinitely.

```
[I 17:32:35.114 NotebookApp] Adapting to protocol v5.1 for kernel 3a797793-0a5d-4157-a1f1-048486a13d67
/home/tiago.motta/.virtualenvs/teste1/lib/python3.6/site-packages/notebook/base/zmqhandlers.py:284: RuntimeWarning: coroutine 'WebSocketHandler.get' was never awaited
  super(AuthenticatedZMQStreamHandler, self).get(*args, **kwargs)
[W 17:32:36.131 NotebookApp] Replacing stale connection: 3a797793-0a5d-4157-a1f1-048486a13d67:fbd7c223acee449386fcc14885dfa03f
```

How to reproduce:

```
$ pip install jupyter
$ pip freeze | grep tornado
tornado==6.0
$ jupyter notebook
```

Workaround:

```
pip install tornado==5.1.1
```

**Comment by JohnCrickett** [comment]
I see the same issue with Python 3.6.7 on Windows.

**Comment by jasongrout** [comment]
My guess is this is the incompatibility with tornado 6 a lot of people have been experiencing. See https://github.com/jupyter/notebook/issues/4439 and https://github.com/jupyter/notebook/pull/4449. The solution would be either upgrading notebook to version 5.7.5, or downgrading tornado to version 5.1.1.

**Comment by minrk** [comment]
Thanks for tracking these @jasongrout! Closing, since this is fixed in 5.7.5
